nautobot/docs/user-guide/platform-functionality/template-filters.md

```diff
@@ -225,10 +225,19 @@ Render a dictionary as formatted JSON.
 +/- 2.2.0
     Unless `syntax_highlight=False` is specified, the returned string will be wrapped in a `<code class="language-json>` HTML tag to flag it for syntax highlighting by highlight.js.
 
++++ 2.4.0
+    Added option for pretty print the output: `pretty_print=False`, default is False. If True the output will be wrapped in `<pre>` tags.
```

Comment:
This is existing functionality isn't it?

Reply:
Yes, but I wanted to mention that in the documentation.

Reply:
Ok, then let's change `2.4.0` to `2.3.0` above, as that's the version where this functionality was first added.